types/tsc-watch/index.d.ts

```diff
@@ -0,0 +1,6 @@
+// Type definitions for tsc-watch 4.2
+// Project: https://github.com/gilamran/tsc-watch#readme
+// Definitions by: Piotr Błażejewicz <https://github.com/peterblazejewicz>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import './lib/client';
```

Comment:
I recommend just using `export {};`, since that import statement isn't going to have any effect.

Reply:
@uniqueiniquity thank you! That bothered me how to correctly reflect native module here. Rebased/amended!